emails/views.py

```diff
@@ -799,23 +808,24 @@ def _convert_to_forwarded_email(
     sample_trackers: bool,
     remove_level_one_trackers: bool,
     now: datetime | None = None,
-) -> tuple[EmailMessage, int, bool, bool]:
+) -> tuple[EmailMessage, EmailForwardingIssues, int, bool, bool]:
     """
     Convert an email (as bytes) to a forwarded email.
 
     Return is a tuple:
     - email - The forwarded email
+    - issues - Any detected issues in conversion
     - level_one_trackers_removed (int) - Number of trackers removed
     - has_html - True if the email has an HTML representation
     - has_text - True if the email has a plain text representation
     """
-    email = message_from_bytes(incoming_email_bytes, policy=policy.default)
+    email = message_from_bytes(incoming_email_bytes, policy=relay_policy)
     # python/typeshed issue 2418
     # The Python 3.2 default was Message, 3.6 uses policy.message_factory, and
     # policy.default.message_factory is EmailMessage
     assert isinstance(email, EmailMessage)
 
-    _replace_headers(email, headers)
+    header_issues = _replace_headers(email, headers)
```

Comment:
suggestion (non-blocking): code-smell: this function changes variable state. 😞 Could we change it to return, and change the calling code to handle, a tuple of `email_with_replaced_headers, header_issues` ?
```suggestion
    email_with_replaced_headers, header_issues = _replace_headers(email, headers)
```

Reply:
Oh, good point. I keep forgetting the memory concern. I'll try to remember, but we could also throw a code comment into the functions, or before the function calls, that say why we're changing the object in-memory.